h2o-genmodel/src/main/java/hex/genmodel/algos/tree/SharedTreeNode.java

```diff
@@ -609,4 +610,15 @@ public final int getRightChildIndex() {
     return rightChild != null ? rightChild.internalId : -1;
   }
 
+    public float getGain(boolean useSEforGain) {
+      if (useSEforGain) {
```

Comment:
where is `useSEforGain` defined? I dont think `useSquaredErrorForGain` is too long, but much more readable

Reply:
it depends on whether you call GBMModel.getFeatureInteractions(...) or XGBoostModel.getFeatureInteractions(). for XGB, gain is prepared in boosterbyte and set in XGBoostModel.constructSubgraph(...) so useSquaredErrorForGain = false, for GBM it's not prepared and it has to be calculated from squared error, so useSquaredErrorForGain = true

Reply:
sorry, stupid question, I missed the method arg `boolean useSEforGain`
knit: would also use the long name here